### PR TITLE
Fix uploading text position

### PR DESCRIPTION
**Pull request by skjnldsv**
| Before | After |
|:---------:|:------:|
|||
|||

### CODE REVIEW

**Review by Pytal — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]
Definitely an improvement so green from my side, but wouldn't it be better to vertically align the two lines with the row, like we do in `listitem` and `listitemicon`? 

Screenshot 2021-12-29 at 11 01 18

File: apps/files/css/files.scss [hunk 1]
```diff
@@ -460,13 +455,22 @@ table td.filename .nametext .innernametext {
...
+	// align with file name
+	padding-left: 1px;
 	font-size: 11px;
-	opacity: .5;
+	// double the font size
+	line-height: 22px;
+	color: var(--color-text-maxcontrast);
+	text-overflow: ellipsis;
+	white-space: nowrap;
 }
```
Comment (szaimen) [inline apps/files/css/files.scss:474]:
```suggestion
	overflow: hidden;
}
```

File: apps/files/css/files.scss [hunk 2]
```diff
@@ -460,13 +455,22 @@ table td.filename .nametext .innernametext {
...
-	margin-top: 5px;
+	// checkbox width
+	margin-left: 50px;
+	left: 0;
+	bottom: 0;
 	height: 20px;
-	padding: 10px 0;
+	padding: 0 4px;
+	// align with file name
+	padding-left: 1px;
```
Comment (szaimen) [inline apps/files/css/files.scss:467]:
```suggestion
	padding-left: 0px;
```

**Review by szaimen — APPROVED** [review]
lgtm otherwise

File: apps/files/css/files.scss (hunk 2 above)
Comment (skjnldsv) [reply apps/files/css/files.scss:467]: No, otherwise the font size difference makes it weird :thinking:

**Comment by skjnldsv** [comment]
> Definitely an improvement so green from my side, but wouldn't it be better to vertically align the two lines with the row, like we do in `listitem` and `listitemicon`?

Well, that ain't gonna be a fix for now considering all the mess there is with absolute positioning.

File: apps/files/css/files.scss (hunk 1 above)
Comment (skjnldsv) [reply apps/files/css/files.scss:474]: Yeah, actually it's broken because of the absolute. It has no effect, maybe for a later rewrite :shrug:

Comment (szaimen) [reply apps/files/css/files.scss:474]: But it works in grid view...

**Pull request merged**